### ISSUE TITLE
Compatibility of Reading GraphML and Writing GML

### ISSUE THREAD

**Issue by hcars** [networkx/networkx#4609]
When I read in a graphml file with read_graphml, an error is produced when writing the same graph to GML with write_gml.

### Current Behavior
write_graphml reads a graph into NetworkX. An error is produced when calling write_gml with the graph.
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<decorator-gen-436>", line 2, in write_gml
  File "/apps/software/standard/core/anaconda/5.2.0-py3.6/lib/python3.6/site-packages/networkx/utils/decorators.py", line 227, in _open_file
    result = func_to_be_decorated(*new_args, **kwargs)
  File "/apps/software/standard/core/anaconda/5.2.0-py3.6/lib/python3.6/site-packages/networkx/readwrite/gml.py", line 810, in write_gml
    for line in generate_gml(G, stringizer):
  File "/apps/software/standard/core/anaconda/5.2.0-py3.6/lib/python3.6/site-packages/networkx/readwrite/gml.py", line 717, in generate_gml
    for line in stringize(attr, value, ignored_keys, '  '):
  File "/apps/software/standard/core/anaconda/5.2.0-py3.6/lib/python3.6/site-packages/networkx/readwrite/gml.py", line 657, in stringize
    raise NetworkXError('%r is not a valid key' % (key,))
networkx.exception.NetworkXError: 'node_default' is not a valid key
```

### Expected Behavior
The graph should be written out to a GML file.

### Steps to Reproduce
Put the following graphml in a file:
```
<?xml version='1.0' encoding='utf-8'?>
<graphml xmlns="http://graphml.graphdrawing.org/xmlns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns http://graphml.graphdrawing.org/xmlns/1.0/graphml.xsd">
  <graph edgedefault="undirected">
    <node id="1" />
    <node id="2" />
    <node id="3" />
    <edge source="1" target="2" />
    <edge source="1" target="3" />
  </graph>
</graphml>
```
Run the following:
```
>>> import networkx as nx
>>> G = nx.read_graphml("name_of_graphml_file")
>>> nx.write_gml(G,"test.gml")
```

### Environment

Python version: 3.7.7
NetworkX version: 2.4

**Comment by MridulS** [comment]
This is fixed in NetworkX 2.5 https://github.com/networkx/networkx/pull/3841.
Can you update networkx locally and test it out?
```
$ pip install networkx==2.5
```

**Comment by hcars** [comment]
Yep, I just checked 2.5 fixed it. Thanks. You may want to add this to the release notes as I didn't see it there.
